### DISCUSSION TITLE
Space for GitHub Actions feedback?

### DISCUSSION THREAD

**Discussion by pllim** [community/community#7914] [Actions] [answered]
I do not see any dedicated space for feedback on GitHub Actions. Is there one? Will there be one?

**Comment by michellemerrill** [comment, marked as answer]
👋🏼 @pllim - We just opened up our GitHub Actions category today: https://github.com/github/feedback/discussions/8473

**Reply by dentuzhik** [reply]
Nice!
Is this a new default or will team still monitor https://github.community/c/code-to-cloud/github-actions/41?
Now that there are two places to post, which one is likelier to get more attention?

**Reply by dentuzhik** [reply]
Found my answer in this announcement:
https://github.com/github/feedback/discussions/8473
